codecov.yml

```diff
@@ -10,15 +10,34 @@ coverage:
         target: 78%
         # Whole-project test coverage is allowed to drop up to 5%. (For situtations where we delete code with full coverage)
         threshold: 5%
+        flags:
+          - back-end
 
       front-end:
         target: 35%
         threshold: 5%
+        flags:
+          - front-end
 
     patch:
       back-end:
         # Changes must have at least 75% test coverage (by line)
         target: 75%
+        flags:
+          - back-end
 
       front-end:
         target: 35%
+        flags:
+          - front-end
+
+flags:
+  back-end:
+    paths:
+      - enterprise/backend
+      - shared/src
+      - src/metabase
+
+  front-end:
+    paths:
+      - frontend
```

Comment:
Do we need to add `enterprise/frontend` as well?

Reply:
I can't seem to find any tests (`enterprise/frontend/test`), but you're right, probably safe to just add it now.